recipes/linux-headers/linux-headers.inc

```diff
@@ -4,7 +4,7 @@ inherit kernel-headers
 RECIPE_TYPES = "machine sdk"
 
 require conf/fetch/kernelorg.conf
-SRC_URI = "${KERNELORG_MIRROR}/linux/kernel/v${KERNEL_VERSION_PATCHLEVEL}/linux-${PV}.tar.bz2"
+SRC_URI = "${KERNELORG_MIRROR}/linux/kernel/v${@'${PV}'.split('.')[0]+'.x'}/linux-${PV}.tar.xz"
```

Comment:
Ok, but correct me if I am wrong.  We are breaking existing pre 3.0 linux recipes with this?  I am not objecting, as the fix is clearly trivial, but it needs to go in the release notes for 7.0.